uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/common/utils/UnitFormatUtils.kt

```diff
@@ -112,3 +114,29 @@ object Distance {
         }
     }
 }
+
+object Time {
```

Comment:
tieto objecty tu v Kotline moc zmysel nemaju... ako nie je to zle a chapem, ze je to kvoli rozdeleniu, ale myslim ze spravne by to malo byt ako top-level funkcie... Ten performance rozdiel realne asi bude viac menej ziadny, len toto v podstate vytvori staticky singleton objekt, ktory ale realne nepotrebujes